### PR TITLE
fix: indent and commas

### PR DESCRIPTION
**Pull request by o0Ignition0o**
While formatting .graphql files, my linter added commas and newlines around. Although this is allowed in the spec, and the parser handles it correctly, we weren't able to handle it when gathering endpoint names and urls.

This commits fixes this, and adds various indents and commas in the routing_urls test.

### CODE REVIEW

File: crates/apollo-router-core/src/schema.rs [hunk 1]
```diff
@@ -117,11 +117,21 @@ impl std::str::FromStr for Schema {
...
+                                                                .map(|token| {
+                                                                    token
+                                                                        .text()
+                                                                        .trim()
+                                                                        .trim_start_matches('"')
+                                                                        .trim_end_matches('"')
+                                                                        .trim()
+                                                                        .to_string()
+                                                                })
                                                         });
```
Comment (lrlna) [inline crates/apollo-router-core/src/schema.rs:135]: if you update to the latest `apollo-rs` commit you can get the STRING_VALUE as rust's `String` type:

```rust
if let ast::Value::StringValue(val) =
    argument.value().expect("Cannot get argument value.")
{
    let s: String = val.into();
}
```

It should help you eliminate this green node manipulation happening here.

Comment (lrlna) [reply crates/apollo-router-core/src/schema.rs:135]: ^ It is still preferable that you match on the variants of `argument.value()`, as not all variants implement casting to native rust types (they are not likely to in the future either).

Comment (o0Ignition0o) [reply crates/apollo-router-core/src/schema.rs]: it works! 🙌 thank you! 🚀

**Review by lrlna — APPROVED** [review]
ahhh very nice! i like the way this looks.

**Review by cecton — APPROVED** [review]
LGTM though I don't have opinion on styling XD

**Pull request merged**